### PR TITLE
fix(coredumps): ignore bash coredump

### PR DESCRIPTION
**Pull request by fruch**
reinstate eb395ebc0151c5b718655ed959feee315c7719bb, but without
removing the coredump just ignoring it

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/cluster.py [hunk 1]
```diff
@@ -993,6 +993,13 @@ def _get_core_pids(self):
         #     Tue 2020-01-14 16:18:56 +07    6321  1000  1000   3 present   /usr/bin/scylla
         #     Tue 2020-01-14 16:31:39 +07   18554  1000  1000   3 present   /usr/bin/scylla
         for line in result.splitlines():
+            if "bash" in line:
```
Comment (roydahan) [inline sdcm/cluster.py:996]: isn't it too general? just "bash" would be enough?

Comment (fruch) [reply sdcm/cluster.py:996]: the only string in those line, is the name of the process.
we mostly care about scylla coredumps, and it won't filter those

**Comment by bentsi** [comment]
@fruch please check unit-tests

File: sdcm/cluster.py (hunk 1 above)
Comment (amoskong) [reply sdcm/cluster.py:996]: Can we match the full path of bash?

'/usr/bin/bash' in line

```
$ sudo coredumpctl --no-pager --no-legend
Fri 2020-07-10 01:11:16 UTC    1796  1004  1005  11 * /usr/bin/bash
```

**Pull request merged**